tools/ci_build/build.py

```diff
@@ -1117,9 +1121,7 @@ def generate_build_tree(
             "-DCMAKE_TOOLCHAIN_FILE=" + os.path.join(source_dir, "cmake", "riscv64.toolchain.cmake"),
         ]
 
-    # By default on Windows we currently support only cross compiling for ARM/ARM64
-    # (no native compilation supported through this script).
-    if args.arm64 or args.arm64ec or args.arm:
+    if is_arm_cross_compile(args):
```

Comment:
So, your this change is mainly for adding the support for building ARM64EC binaries on ARM64 machines?